src/web.ts

```
@@ -65,64 +92,194 @@ export class NativeAudioWeb extends WebPlugin implements NativeAudio {
+      audio.preload = 'metadata';
+      audio.addEventListener('loadedmetadata', () => {
+        resolve();
+      });
+      audio.addEventListener('error', () => reject('Error loading audio file'));
+
+      if (options.volume) {
+        audio.volume = options.volume;
+        this.initialVolume = options.volume;
+      }
+      NativeAudioWeb.AUDIO_ASSET_BY_ASSET_ID.set(options.assetId, new AudioAsset(audio));
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)